src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+// Navigation ------------- //
+
+$height-nav-secondary: 5rem;
+$width-nav-sidebar:    250px;
+
+
+.usa-header {
+  width: 100%;
+  z-index: 1;
```

Comment:
Do you think we should add a system now? I think I remember seeing a couple of z-index values in the header file.

We could also just do variables in this header file to start, which would potentially make it easier to move them to site wide in another refactor.